docs/develop/sso-in-office-add-ins.md

```diff
@@ -222,10 +222,10 @@ There are some small, but important differences in using SSO in an Outlook add-i
 
 ### getAccessToken
 
-The OfficeRuntime [Auth](/javascript/api/office/officeruntime.auth) namespace, `OfficeRuntime.auth`, provides a method, `getAccessToken` that enables the Office host to obtain an access token to the add-in's web application. Indirectly, this also enables the add-in to access the signed-in user's Microsoft Graph data without requiring the user to sign in a second time.
+The OfficeRuntime [Auth](/javascript/api/office-runtime/officeruntime.auth) namespace, `OfficeRuntime.auth`, provides a method, `getAccessToken` that enables the Office host to obtain an access token to the add-in's web application. Indirectly, this also enables the add-in to access the signed-in user's Microsoft Graph data without requiring the user to sign in a second time.
```

Comment:
should `OfficeRuntime.auth` instead be `OfficeRuntime.Auth` (capital A)?

![image](https://user-images.githubusercontent.com/3019529/68719488-06aa1d80-0561-11ea-9050-207230efe7a0.png)